eitprocessing/mixins/slicing.py

```diff
@@ -119,11 +113,15 @@ def __getitem__(self, key: slice | int | float):
         if isinstance(key, slice):
             if key.step:
                 raise ValueError("Can't slice by time using specific step sizes.")
-            start_value = key.start or self.obj.time[0]
-            end_value = key.stop or np.inf
+            if start_value is None:
+                start_value = self.obj.time[0]
+            if end_value is None:
+                end_value = np.inf
             return self.obj.select_by_time(start_value, end_value)
+
         if isinstance(key, (int, float)):
             return self.obj.select_by_time(start=key, end=key, end_inclusive=True)
```

Comment:
This option (to just select a single timepoint, rather than a range) requires
to have the end included. Alternatively, we could make a separate method
`select_single_timepoints()` or similar, but it would share most code with `select_by_time()`.